src/cli/package.rs

```diff
@@ -119,7 +125,48 @@ impl Packager {
 
                 main_tree.insert(file_name.clone(), encoded_content.into());
             } else if node.is_dir() {
-                if let Some(build_config) = node
+                // a folder within this folder has a .build in it, meaning this node
+                // should build the json and insert it for this zome
+                if let Some(dir_with_code) = node
+                    .read_dir()?
+                    .filter(|e| e.is_ok())
+                    .map(|e| e.unwrap().path())
+                    .filter(|path| {
+                        let maybe_metadata = fs::metadata(path);
+                        match maybe_metadata {
+                            Ok(metadata) => metadata.is_dir(),
+                            Err(_) => false,
+                        }
+                    })
+                    .find(|path| path.join(BUILD_CONFIG_FILE_NAME).exists())
+                {
+                    meta_tree.insert(file_name.clone(), META_DIR_ID.into());
+
+                    let build = Build::from_file(dir_with_code.join(BUILD_CONFIG_FILE_NAME))?;
+
+                    let wasm = build.run(&dir_with_code)?;
+
+                    let wasm_binary = base64::decode(&wasm)?;
+
+                    let context= test_context("HC");
+                    let call_result = ribosome::run_dna(
+                        &"HC",
+                        context,
+                        wasm_binary,
+                        &ZomeFnCall::new(&"", &"", &"__hdk_get_json_definition", &""),
+                        Some(String::from("{}").into_bytes())
```

Comment:
```suggestion
                        Some("{}".as_bytes().to_vec())
```
So we don't have to allocate a complete string for this simple manoeuvre.